apps/web/src/components/(teams)/dialogs/invite-team-member-dialog.tsx

```diff
@@ -70,41 +74,61 @@ export const InviteTeamMembersDialog = ({
   ...props
 }: InviteTeamMembersDialogProps) => {
   const [open, setOpen] = useState(false);
+  const uploadRef = useRef<HTMLInputElement | null>(null);
+  const [invitationType, setInvitationType] = useState('INDIVIDIUAL');
 
   const { toast } = useToast();
 
   const form = useForm<TInviteTeamMembersFormSchema>({
     resolver: zodResolver(ZInviteTeamMembersFormSchema),
     defaultValues: {
-      invitations: [
-        {
-          email: '',
-          role: TeamMemberRole.MEMBER,
-        },
-      ],
+      email: '' || [''],
+      role: TeamMemberRole.MEMBER,
     },
   });
 
-  const {
-    append: appendTeamMemberInvite,
-    fields: teamMemberInvites,
-    remove: removeTeamMemberInvite,
-  } = useFieldArray({
-    control: form.control,
-    name: 'invitations',
-  });
-
   const { mutateAsync: createTeamMemberInvites } = trpc.team.createTeamMemberInvites.useMutation();
 
-  const onAddTeamMemberInvite = () => {
-    appendTeamMemberInvite({
-      email: '',
-      role: TeamMemberRole.MEMBER,
-    });
+  const handleFileUpload = (e: React.ChangeEvent<HTMLInputElement>) => {
+    if (!e.target.files?.length) {
+      return;
+    }
+    const file = e.target.files[0];
+
+    if (file) {
+      const reader = new FileReader();
+      const emailRegex = /^([A-Z0-9_+-]+\.?)*[A-Z0-9_+-]@([A-Z0-9][A-Z0-9-]*\.)+[A-Z]{2,}$/i;
```

Comment:
Could we not use zod here?

Reply:
Yeah, we are also using Zod here. We can remove this.